packages/web/src/pages/rewards-page/config.tsx

```diff
@@ -97,42 +97,27 @@ type WebChallengeInfo = {
 }
 
 const webChallengesConfig: Record<ChallengeRewardID, WebChallengeInfo> = {
-  referrals: {
-    icon: <i className='emoji large incoming-envelope' />
-  },
-  [ChallengeName.Referrals]: {
-    icon: <i className='emoji large incoming-envelope' />
-  },
-  'ref-v': {
-    icon: <i className='emoji large incoming-envelope' />
-  },
-  [ChallengeName.ReferralsVerified]: {
-    icon: <i className='emoji large incoming-envelope' />
-  },
-  referred: {
-    icon: <i className='emoji large love-letter' />
-  },
-  [ChallengeName.Referred]: {
-    icon: <i className='emoji large love-letter' />
-  },
+  referrals: {},
```

Comment:
can the changes in this file be explained a bit more? either in PR description or here. not seeing where this stuff has been replaced. is that intentional?

Reply:
yeah what was the idea here @dharit-tan ?

Reply:
this was just removing the icons since the new designs don't have any icons in the modal/tile headers at all